starter/build.gradle

```diff
@@ -1,6 +1,9 @@
 dependencies {
-    implementation 'org.springdoc:springdoc-openapi-security:1.2.32'
+    compile 'org.springdoc:springdoc-openapi-security:1.2.32'
```

Comment:
can you move it to dep management section of the parent build gradle? Also if you use implementation, then better to stick with it instead of compile

Reply:
It has sub dependencies to the swagger v3 annotations which might be used by our microservices that why i need to change it to compile.

Reply:
ah, ok, then you can use api keyword

Reply:
https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle